src/tiled/propertybrowser.h

```diff
@@ -134,7 +135,7 @@ private slots:
         FlippingProperty,
         DrawOrderProperty,
         FileNameProperty,
-        TileOffsetProperty,
+        TilesetTileOffsetProperty,
```

Comment:
On second thought you don't need this one, you can us `TileOffsetProperty` for both the tile and the tileset, since they are mutually exclusive in the properties view.